pkgs/applications/misc/memo/default.nix

```diff
@@ -15,6 +15,9 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/{bin,share/man/man1,share/bash-completion/completions}
+    substituteInPlace memo \
+      --replace "ack " "${ag}/bin/ag " \
+      --replace "tree" "${tree}/bin/tree "
```

Comment:
The match contains no space yet the substitution does. I expect this makes no practical difference, but just in case ... (or perhaps it's intentional?)

Reply:
No it was not intentional. I fixed it.